src/pages/Swap/TaxTooltipBody.tsx

```diff
@@ -0,0 +1,34 @@
+import { Trans } from '@lingui/macro'
+import styled from 'styled-components'
+import { ThemedText } from 'theme'
+
+const Divider = styled.div`
+  width: 100%;
+  height: 1px;
+  border-width: 0;
+  margin: 12px 0;
+  background-color: ${({ theme }) => theme.surface3};
+`
+
+export function OutputTaxTooltipBody({ currencySymbol }: { currencySymbol?: string }) {
+  return (
+    <>
+      <ThemedText.SubHeaderSmall color="textPrimary">
+        <Trans>Exact input only</Trans>
```

Comment:
Is "Exact input" used colloquially? I worry this is only used internally and outside users may get confused. Maybe "Input currency only"  or "Input amount only"

Reply:
Not sure, but this came from cal & fred 🤷